RELEASE.md

```diff
@@ -9,46 +9,31 @@ necessary.
 
 Make sure there aren't any pending issues before tagging a new release.
 
-## Release commit
+## Run release script
 
-### Change version in `Dockerfile`
+The release script is used to make the necessary changes to enable tagging
+version `vX.Y.Z`:
 
-The image version specified in [`Dockerfile`](./Dockerfile) should be `vX.Y.Z`.
-
-### Change version in CHANGES
-
-The "Unreleased" version in [CHANGES](./CHANGES.md) should be changed to
-`vX.Y.Z`, and a new "Unreleased" header should be added.
-
-### Add release description to CHANGES
-
-This description should answer the "who needs this update?" question.
+```
+$ ./make-release.sh vX.Y.Z
+```
 
-### Create release commit
+The script will create a separate branch, where it will make the necessary
+changes and prompt you for any further required information.
 
-This commit should contain only the above change, and be titled `Release
-version vX.Y.Z.`.
 
```

Comment:
Nit: we should remove one of the newlines surrounding this paragraph. Otherwise, it ends up two empty lines in a sequel.